### PR TITLE
DrawableRoom design update

### PR DESCRIPTION
**Pull request by DrabWeb**
screen shot 2017-06-24 at 5 57 15 am

Didn't have the designs before so assumed alot wrong. Also includes general cleanup and shared code of/between `DrawableRoom` and `RoomInspector`.

### CODE REVIEW

File: osu.Game/Screens/Multiplayer/DrawableRoom.cs [hunk 1]
```diff
@@ -59,24 +68,36 @@ public DrawableRoom(Room room)
...
+                        {
+                            RelativeSizeAxes = Axes.Both,
+                        },
+                    },
                 },
                 new Container
                 {
                     RelativeSizeAxes = Axes.Both,
-                    Padding = new MarginPadding { Top = content_padding, Bottom = content_padding, Left = Height + content_padding * 2, Right = content_padding },
+                    Padding = new MarginPadding { Top = content_padding, Bottom = content_padding, Left = side_strip_width + cover_width + content_padding, Right = content_padding },
```
Comment (peppy) [inline osu.Game/Screens/Multiplayer/DrawableRoom.cs]: i still think cases like this should be split into newlines per property setter.

File: osu.Game/Screens/Multiplayer/DrawableRoom.cs [hunk 2]
```diff
@@ -97,7 +97,13 @@ public DrawableRoom(Room room)
                 new Container
                 {
                     RelativeSizeAxes = Axes.Both,
-                    Padding = new MarginPadding { Top = content_padding, Bottom = content_padding, Left = side_strip_width + cover_width + content_padding, Right = content_padding },
+                    Padding = new MarginPadding
+                    {
+                        Top = content_padding,
```
Comment (EVAST9919) [inline osu.Game/Screens/Multiplayer/DrawableRoom.cs]: Use Vertical instead of Top and Bottom

**Review by EVAST9919 — CHANGES_REQUESTED** [review]

**Comment by peppy** [comment]
https://puu.sh/wtOMf/91cbedae7f.gif this probably needs to be fixed. if they are going to animate, they should only do so on the horizontal axis

**Comment by peppy** [comment]
Looks good. Not going to nit-pick on this one since it's not being actively used yet. Will probably require some changes once we slot it all in.

**Review by peppy — APPROVED** [review]

**Pull request merged**